packages/yigim-client/src/components/probmaking.js

```diff
@@ -44,25 +64,33 @@ class ProbMaking extends Component{
         })
     }
     render(){
-        var answers_each=[];
         var data=this.state.QnAExSets[this.probNumber];
-        var question=data.question
-        var j=0;
-       /* const array=this.state.QnAExSets.map(
-            item => item.id===0
-            ? item
-            : ({...item, answer: answers_each})
-        )*/
-        while (j<data.answer.length){
-            answers_each.push(
-                data.answer[j]
-            )
-            j=j+1;
-        }
+        var question=data.question;
+        var show_question=question;
+        var answers_each=data.answer.map(string => string)

```

Comment:
```suggestion
        var answers_each=data.answer
```
와 같습니다. 불필요한 map 사용입니다.